package.json

```
@@ -47,8 +47,10 @@
     "react-router-dom": "^6.26.1"
   },
   "scripts": {
-    "dev:start": "REACT_APP_JAM_DEV_MODE=true npm start",
-    "dev:start:secondary": "PORT=3001 JAM_BACKEND=jam-standalone JAM_API_PORT=29080 npm run dev:start",
+    "dev:start": "echo 'Deprecated command will be removed soon. Please use `npm run dev` instead.' && npm run dev",
```

```suggestion
    "dev:start": "echo 'Deprecated command will be removed soon. Please use `npm run dev` instead.'",
```
this could be removed maybe?
if you don't do install first and just run npm run dev then the && in front will not work ofc..
feel free to ignore